webapp/components/ContributionForm/ContributionForm.vue

```diff
@@ -18,7 +19,23 @@
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-12-04T19:03:16Z" title="Wednesday, December 4th 2019, 8:03:16 pm +01:00">Dec 4, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -7,14 +7,57 @@
     @submit="submit"
   >
     <template slot-scope="{ errors }">
-      <hc-teaser-image :contribution="contribution" @addTeaserImage="addTeaserImage">
+      <hc-teaser-image
+        :contribution="contribution"
+        @addTeaserImage="addTeaserImage"
+        :class="{ 'images-set-blur': checkedBlur }"
```


```suggestion
        :class="{ 'images-set-blur': blurImage }"
```